File: 1967-number-of-strings-that-appear-as-substrings-in-word/1967-number-of-strings-that-appear-as-substrings-in-word.cpp
class Solution {
public:
    int numOfStrings(vector<string>& patterns, string word) {
        int subs = 0;
        for(string &s: patterns){
            if(word.find(s) != string::npos)
                subs++;
        }
        return subs;
    }
};

File: 1967-number-of-strings-that-appear-as-substrings-in-word/1967-number-of-strings-that-appear-as-substrings-in-word.rs
impl Solution {
    pub fn num_of_strings(patterns: Vec<String>, word: String) -> i32 {
        let mut substring:i32 = 0;
        
        for s in patterns {
            if(word.contains(&s)) {
                substring += 1;
            }
        }

        return substring;
    }
}